client/app/redux/actions/FavoritesActions.js

```diff
@@ -25,37 +15,40 @@ export function getFavorites(userID) {
         }
     }
 
-    // -------- The Failure Action --------
-    // Called in the catch() block so Redux knows error occurred + what to do.
-    // Maybe set some stuff back to 'undefined' and "isLoading" back to false
     const failure = (errorMessage) => {
         return {
             type: types.GET_FAVORITES_FAILURE,
             errorMessage: errorMessage
         }
     }
 
-    // -------- The actual THUNK! --------
-    // dispatch() is the function that executes
-    // one of the above Redux actions.
     return async (dispatch) => {
         dispatch(request()); // tell Redux we're about to make that request
         try {
-            // TODO: change this to get favorites
-            // const favorites = await get("/api/menus", { location: locationID });
-            const favorites = [
-                {
-                    name: 'Grilled Cheese',
-                    itemID: 1,
-                },
-                {
-                    name: 'Cucumber Pizza',
-                    itemID: 2,
-                },
-            ]
-            dispatch(success(favorites)); // If successfull, dispatch it to Redux
+            const favorites = await get("/api/favorites", { 
+                token: expoToken
+             });
+            dispatch(success(favorites)); // If successful, dispatch it to Redux
         } catch (e) {
             dispatch(failure(e.message));
         }
     }
+}
+
+export function addFavorite(menuItemID) {
+    return {
+        type: types.ADD_FAVORITE,
+        payload: {
```

Comment:
Don't we need menuitemid and name for addFavorite?

Reply:
Yep, we do now. I'll add that!